cartridge/shop/models.py

```diff
@@ -129,6 +141,8 @@ def save(self, *args, **kwargs):
         if updating and not settings.SHOP_USE_VARIATIONS:
             default = self.variations.get(default=True)
             self.copy_price_fields_to(default)
+        elif hasattr(self, 'content_model'):
+            self.content_model = self._meta.object_name.lower()
```

Comment:
> If we ever need to implement logic in ContentTyped that distinguishes between default and non-default content models, we'll have an easy way to check.

Just this.

It's a more consistent use of the field: if we leave this `None` for default `Product`s, the field accurately represents *the subclass* for each instance. If we don't, it represents *the subclass or if there is none then the base class*, which we then have to unpack every time we need to make that distinction.